samladmin/locales/en_US/admin.UTF-8.properties

```diff
@@ -71,6 +71,10 @@ attrmapping.title=Configuration of SAML accounts
 attrmapping.error.not.set=Configuration of SAML attributes is not completed
 attrmapping.form.login.label=SAML attribute containing the id/login of the user
 attrmapping.form.attrsgroup.label=list of SAML attributes corresponding to each user properties
+attrmapping.form.groupsfromsaml.label=SAML provides user groups
+attrmapping.form.groupsfromsamlattr.label=Groups attribute
+attrmapping.form.groupsfromsamlsep.label=Groups separator
+attrmapping.form.groupsfromsamlprefix.label=Groups prefix
```

Comment:
We should explain what is the goal of the prefix. It is not clear to understand for the user, if this is a prefix that is added to the given groups, or that is removed, or if it has an other goal.

Reply:
I have added help strings.